x/concentrated-liquidity/incentives.go

```diff
@@ -381,37 +368,14 @@ func (k Keeper) initOrUpdatePositionUptime(ctx sdk.Context, poolId uint64, posit
 		if freezeDuration >= uptime {
 			curUptimeAccum := uptimeAccumulators[uptimeIndex]
 
-			// If a record does not exist for this uptime accumulator, create a new position.
-			// Otherwise, add to existing record.
-			recordExists, err := curUptimeAccum.HasPosition(positionName)
```

Comment:
yup just added

Reply:
hrmm now that i'm thinking about this, i think it makes sense to refactor the entire position function `initOrUpdatePosition` -> `initPosition` and remove update. What do you think?

Reply:
upon further inspection i think returning an error if !recordExist is not a good idea rn as this will break lots of other key functions. For instance, `updatePosition` while calling `withdrawPosition` fails because the position already exist but that we should call `updatePosition` regardless  and not call `initUptime` 

i could refactor the entire flow of init  and update position, but i think thats out of scope for this PR. Will create an issue to track this, meanwhile i propose we keep the logic as is without throwing an error